lumen/transforms/sql.py

```diff
@@ -90,3 +90,41 @@ def apply(self, sql_in):
         return Template(template, trim_blocks=True, lstrip_blocks=True).render(
             limit=self.limit, sql_in=sql_in
         )
+
+
+class SQLDistinct(SQLTransform):
```

Comment:
I like this framework and will be nice to have these as SQLTransforms in general

Reply:
Yep, I'm very happy with `SQLTransform` thanks for getting the ball rolling on those.